python/social_bot/envs/play_ground.py

```diff
@@ -260,11 +260,13 @@ def main():
     import matplotlib.pyplot as plt
```

Comment:
There is a separate play_ground_test.py, maybe we can remove this test?

Reply:
It's till useful when we want render, test specific case, or run for a long time to test memory growth or something. Let's keep it.